internal/mode/static/state/change_processor.go

```diff
@@ -33,6 +32,16 @@ const (
 		"webhook is running correctly."
 )
 
+const (
+	// NoChange means that nothing changed.
+	NoChange = iota
+	// EndpointsOnly means that only the endpoints changed.
+	// If using NGINX Plus, this update can be done using the API without a reload.
+	EndpointsOnly
```

Comment:
nit: 
```suggestion
	EndpointsOnlyChange
```
for consistency